airflow/contrib/hooks/jenkins_hook.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from airflow.hooks.base_hook import BaseHook
+
+import jenkins
+import logging
+
+
+class JenkinsHook(BaseHook):
+    """
+    Hook to manage connection to jenkins server
+    """
+
+    def __init__(self, conn_id='jenkins_default'):
+        connection = self.get_connection(conn_id)
+        self.connection = connection
+        logging.info('Trying to connect to http://%s:%d' % (connection.host, connection.port))
+        self.jenkins_server = jenkins.Jenkins('http://%s:%d' % (connection.host, connection.port), connection.login, connection._password)
```

Comment:
Please add possibility to use https.

Reply:
Done, I used the field extra of the connection to store a boolean for that.